.travis.yml

```diff
@@ -1,50 +1,16 @@
-language: scala
-scala:
-  - 2.12.8
+services:
+  - docker
 
-os:
-  - osx
-  - linux
-
-osx_image: xcode9.2
-
-before_cache:
-  - if [ "$TRAVIS_OS_NAME" = "osx" ]; then brew cleanup; fi
-
-cache:
-  directories:
-  - $HOME/Library/Caches/Homebrew
-
-addons:
-  homebrew:
-    packages:
-    - gmp
-    - re2
-    - pkgconfig
-    - libtool
-    - protobuf
-    casks:
-    - osxfuse
-    update: true
-  apt:
-    packages:
-    - libre2-dev
-    - python3-pip
-    - makedev
-    - fuse
-    - libfuse-dev
-    - libsqlite3-dev
-    - libgmp-dev
-    - libncurses5-dev
-    - pkg-config
-    - git
-    - g++
-    - gcc
-    - libtool-bin
+before_install:
+  - docker pull sifive/wit:0.10.1
 
 install:
-  - if [ "$TRAVIS_OS_NAME" = "osx" ]; then source continuous-integration/travis/install_deps_osx.sh; fi
-  - if [ "$TRAVIS_OS_NAME" = "linux" ]; then source continuous-integration/travis/install_deps_linux.sh; fi
-
-script: ./continuous-integration/travis/test.sh
-
+  - docker run -d -v `pwd`:`pwd` -w `pwd` -i -t --name wit-host sifive/wit:0.10.1 tail -f /dev/null
+  - docker exec -it wit-host bash -c "git config --global url."https://github.com/".insteadOf 'git@github.com:'"
+  - docker exec -it wit-host bash -c "wit init workspace -a git@github.com:sifive/soc-freedom-sifive.git -a git@github.com:sifive/environment-blockci-sifive.git"
```

Comment:
This will check out `master` of these repos by default. can you copy the version that is checked out by travis into the docker and run wit with `--repo-path` so that we checkout the correct commit of `soc-freedom-sifive`